03/code-monk08/tags.py

```diff
@@ -0,0 +1,42 @@
+from collections import Counter
+from difflib import SequenceMatcher
+from itertools import product
+import re
+
+IDENTICAL = 1.0
+TOP_NUMBER = 10
+RSS_FEED = 'rss.xml'
+SIMILAR = 0.87
+TAG_HTML = re.compile(r'<category>([^<]+)</category>')
+
+
+def get_tags():
+    return TAG_HTML.findall(open(RSS_FEED).read())
+
+
+def get_top_tags(tags):
+    return Counter(tags).most_common(TOP_NUMBER)
+
+
+def get_similarities(tags):
+    result = {}
+    for x, y in product(tags, tags):
+        if x == y or x[0] != y[0]:
+            continue
+        else:
```

Comment:
when you `continue` you probably don't need the `else`

Reply:
Thanks, will take a note of that.